lib/game.rb

```diff
@@ -34,14 +34,66 @@ def message_player_display
   end
   # ============================================
 
-  def fire(declarer, coordinate)
-    if declarer == @user
-      # nested if for user error for firing on coordinate that has been fired on
-      @auto.board.cells[coordinate].fire_upon
-      # require "pry"; binding.pry
-    elsif declarer == @auto
+  def start
+    main_menu
```

Comment:
Love that the start method just immediately has so much accessibility for the user.